src/main/include/subsystems/Shooter.h

```diff
@@ -40,14 +42,18 @@ namespace t34
 
         void RunIntake(const double motor_output);
 
-        void MoveToAngle(const double angle);
+        void MoveToAngleDeg(const double angle);
+
+        void SetMaxSpeedPercent(const double percent);
 
         void Periodic();
 
         void Init();
 
+        void PutTelemetry();
+
         inline double GetTopArmEncoderVal() { return m_arm_encoder_top.GetPosition(); }
-        inline double GetBottomArmEncoderVal() { return m_arm_encoder_top.GetPosition(); }
+        inline double GetBottomArmEncoderVal() { return m_arm_encoder_bottom.GetPosition(); }
 
         inline void RunTopArmMotor(const double motor_output) { m_arm_motor_top.Set(motor_output); }
         inline void RunBottomArmMotor(const double motor_output) { m_arm_motor_bottom.Set(motor_output); }
```

Comment:
When these functions are called in other places, it is difficult to know what they mean. E.g., ...

```cpp
rc->shooter.RunTopArmMotor(0.4);
rc->shooter.RunBottomArmMotor(0.4);
```

Is that 0.4 a fraction/percentage value? If so, it might be clearer to include that in the function name:

```suggestion
        inline void RunTopArmMotorPercent(const double percent) { m_arm_motor_top.Set(percent); }
        inline void RunBottomArmMotorPercent(const double percent) { m_arm_motor_bottom.Set(percent); }
```